xchange-bitcointoyou/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.knowm.xchange</groupId>
+		<artifactId>xchange-parent</artifactId>
+		<version>4.3.3-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>xchange-bitcointoyou</artifactId>
+
+	<name>XChange BitcoinToYou</name>
+	<description>XChange implementation for the BitcoinToYou Exchange</description>
+
+	<url>http://xeiam.com</url>
```

Comment:
in 2015 Xeiam was acquired by Knowm, so this info should be changed.

Reply:
Sorry about that. Done!